app/services/audit_results.rb

```diff
@@ -213,8 +215,12 @@ def log_msg_prefix
     @log_msg_prefix ||= "#{check_name}(#{druid}, #{moab_storage_root&.name})"
   end
 
-  def workflows_msg_prefix
-    @workflows_msg_prefix ||= begin
+  def results_as_string(error_results)
```

Comment:
this refactor in particular will enable easy addition of info to new status_details field in complete_moabs table. 

The other refactors were, in my opinion, clarifications.